test_scripts/fanout

```diff
@@ -40,15 +40,15 @@ pub -p 40010 A B C D E F G H I J K L M N
 pub -p 40011 A B C D E F G H I J K L M N
 pub -p 40012 A B C D E F G H I J K L M N
 
-sleep 1
+sleep 2
 
 expect_pubs_received 196 A B C D E F G H I J K L M N
 
 # Reachability check
 
 pub -p 40005 X
 
-sleep 1
+sleep 2
```

Comment:
not a big deal since it is a test script but why the increase in the sleep time?

Reply:
Extra time is for DTLS handshake.